notifications/message.py

```diff
@@ -40,25 +40,33 @@ class NotificationsMessage(object):
     Initialize the NotificationsMessage object containing various attributes required by steps in the pipeline.
     """
 
-    def __init__(self, name, fields=None, recipients=None):
+    def __init__(
+            self,
+            name,
+            fields=None,
+            recipients=None,
+            expiration_time=None
+    ):
         """
         Initialize the NotificationsMessage object with the object name.
 
         Args:
             name (str): Namespaced string corresponding to the type of notifications message.
             fields (optional): Fields that may be optionally required by steps in the pipeline.
             recipients (optional list): List of recipients that can be specified on creation.
+            expiration_time (datetime): Time of expiry of message.
 
         Attributes:
             uuid (UUID): Universally unique identified for the message.
-            expiration_time (datetime): Time of expiry of message.
             history (list): List of PipelineHistory objects.
             current_step: Indicates the current step in the pipeline.
         """
         self.fields = fields
         self.recipients = recipients
         self.name = name
         self.uuid = uuid.uuid4()
-        self.expiration_time = pytz.utc.localize(datetime.utcnow()) + timedelta(minutes=5)
+        self.expiration_time = expiration_time
+        if self.expiration_time is None:
+            self.expiration_time = pytz.utc.localize(datetime.utcnow()) + timedelta(minutes=5)
```

Comment:
That's right. Time localization needs to be done only at the presentation layer.